### ISSUE TITLE
heroku local:run no longer accepting flags as arguments

### ISSUE THREAD

**Issue by ianstormtaylor** [heroku/heroku-run#15]
I think this must have been related to the `3.0.0` release, but this command no longer works:

```
$ make start
heroku local:run ./node_modules/.bin/nodemon --ignore ./lib ./bin/api

  error: unknown option `--ignore'
```

`run` is getting tripped up by the flags passed to `nodemon`. And using `--` doesn't seem to work to delimit flags either.

**Comment by ransombriggs** [comment]
@ianstormtaylor thanks for the bug report, I am working on a fix, will update once I have shipped something

**Comment by ransombriggs** [comment]
@ianstormtaylor I just shipped a fix for this in `heroku-local@5.0.1`, could you please verify that your command works as expected?

**Comment by ianstormtaylor** [comment]
@ransombriggs yup works perfectly! Thanks for the fix!
